.github/workflows/dependabot-alerts-to-slack.yml

```diff
@@ -0,0 +1,17 @@
+name: 'Check for Dependabot alerts & send them to slack'
+
+on:
+  schedule:
+    - cron: '0 8 * * *' # every day at 8 am
```

Comment:
I'm guessing this is in UTC? We might want to update this and shift this later in the day after we test things out and it works. 👍